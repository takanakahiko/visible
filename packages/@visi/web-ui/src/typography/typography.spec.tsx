import React from 'react';

import { render } from '../render';
import { Typography } from '.';

describe('Typography', () => {
  it('matches snapshot', () => {
    const { container } = render(<Typography variant="body">foo</Typography>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
