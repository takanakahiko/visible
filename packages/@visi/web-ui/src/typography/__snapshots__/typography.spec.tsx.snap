// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography matches snapshot 1`] = `
.c0 {
  margin: 0;
  color: #333333;
  font-size: 14px;
  font-style: normal;
  text-transform: none;
  line-height: 1.6;
}

<p
  class="c0"
>
  foo
</p>
`;
